assets/js/script.js

```diff
@@ -0,0 +1,33 @@
+const hamburgerMenu = document.querySelector('.hamburger-menu');
+hamburgerMenu.setAttribute('class', 'hamburger-menu');
+
+const hamburgerNavMenu = document.querySelector('.hamburger-navmenu');
+const navbar = document.querySelector('.not-navbar');
+
+// Menu when active
+const hamburgerToggle = document.createElement('div');
+hamburgerToggle.classList.add('hamburger-menu.active');
+
+hamburgerMenu.addEventListener('click', () => {
+  if (navbar.classList.contains('hidden')) {
+    navbar.classList.remove('hidden');
+  } else {
+    navbar.classList.add('hidden');
+    // hamburgerMenu.setAttribute('visibility', 'visible');
+  }
+
+  if (hamburgerNavMenu.classList.contains('hidden')) {
+    hamburgerNavMenu.classList.remove('hidden');
+  } else {
+    hamburgerNavMenu.classList.add('hidden');
+  }
+  hamburgerMenu.classList.toggle('active');
+  hamburgerNavMenu.classList.toggle('active');
+  hamburgerNavMenu.classList.toggle('showHamburgerNavMenu');
+});
+
+document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click', () => {
+  hamburgerNavMenu.setAttribute('visibility', 'hidden');
+  navbar.classList.add('hidden');
+  hamburgerMenu.setAttribute('class', 'hamburger-menu');
+}));
```

Comment:
Code redundancy: There is some code redundancy in the `if-else` statements that toggle the `hidden` class. Instead of checking if the element contains the `hidden` class and then adding or removing it, the `toggle` method can be used. This way your code will be DRY(Don't Repeat Yourself), concise, and readable